server/app/views/questiontypes/AddressQuestionRenderer.java

```diff
@@ -128,67 +136,10 @@ protected DivTag renderInputTags(
-        SelectWithLabel.OptionValue.builder().setLabel("MA").setValue("MA").build(),
-        SelectWithLabel.OptionValue.builder().setLabel("MI").setValue("MI").build(),
-        SelectWithLabel.OptionValue.builder().setLabel("MN").setValue("MN").build(),
-        SelectWithLabel.OptionValue.builder().setLabel("MS").setValue("MS").build(),
-        SelectWithLabel.OptionValue.builder().setLabel("MO").setValue("MO").build(),
-        SelectWithLabel.OptionValue.builder().setLabel("MT").setValue("MT").build(),
-        SelectWithLabel.OptionValue.builder().setLabel("NE").setValue("NE").build(),
-        SelectWithLabel.OptionValue.builder().setLabel("NV").setValue("NV").build(),
-        SelectWithLabel.OptionValue.builder().setLabel("NH").setValue("NH").build(),
-        SelectWithLabel.OptionValue.builder().setLabel("NJ").setValue("NJ").build(),
-        SelectWithLabel.OptionValue.builder().setLabel("NM").setValue("NM").build(),
-        SelectWithLabel.OptionValue.builder().setLabel("NY").setValue("NY").build(),
-        SelectWithLabel.OptionValue.builder().setLabel("NC").setValue("NC").build(),
-        SelectWithLabel.OptionValue.builder().setLabel("ND").setValue("ND").build(),
-        SelectWithLabel.OptionValue.builder().setLabel("MP").setValue("MP").build(),
-        SelectWithLabel.OptionValue.builder().setLabel("OH").setValue("OH").build(),
-        SelectWithLabel.OptionValue.builder().setLabel("OK").setValue("OK").build(),
-        SelectWithLabel.OptionValue.builder().setLabel("OR").setValue("OR").build(),
-        SelectWithLabel.OptionValue.builder().setLabel("PW").setValue("PW").build(),
-        SelectWithLabel.OptionValue.builder().setLabel("PA").setValue("PA").build(),
-        SelectWithLabel.OptionValue.builder().setLabel("PR").setValue("PR").build(),
-        SelectWithLabel.OptionValue.builder().setLabel("RI").setValue("RI").build(),
-        SelectWithLabel.OptionValue.builder().setLabel("SC").setValue("SC").build(),
-        SelectWithLabel.OptionValue.builder().setLabel("SD").setValue("SD").build(),
-        SelectWithLabel.OptionValue.builder().setLabel("TN").setValue("TN").build(),
-        SelectWithLabel.OptionValue.builder().setLabel("TX").setValue("TX").build(),
-        SelectWithLabel.OptionValue.builder().setLabel("UT").setValue("UT").build(),
-        SelectWithLabel.OptionValue.builder().setLabel("VT").setValue("VT").build(),
-        SelectWithLabel.OptionValue.builder().setLabel("VI").setValue("VI").build(),
-        SelectWithLabel.OptionValue.builder().setLabel("VA").setValue("VA").build(),
-        SelectWithLabel.OptionValue.builder().setLabel("WA").setValue("WA").build(),
-        SelectWithLabel.OptionValue.builder().setLabel("WV").setValue("WV").build(),
-        SelectWithLabel.OptionValue.builder().setLabel("WI").setValue("WI").build(),
-        SelectWithLabel.OptionValue.builder().setLabel("WY").setValue("WY").build());
+    return STATE_ABBREVIATIONS.stream()
```

Comment:
NB: Should we enforce ordering by calling a sort function or is just relying on people adding them correctly in the list on line 24 adequate?

Reply:
sounds good! added sorting in the creation of the static list